xds/src/main/java/io/grpc/xds/CdsLoadBalancer2.java

```diff
@@ -79,10 +79,16 @@ final class CdsLoadBalancer2 extends LoadBalancer {
   }
 
   @Override
-  public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
+  public boolean acceptResolvedAddresses(ResolvedAddresses resolvedAddresses) {
     if (this.resolvedAddresses != null) {
-      return;
+      return true;
     }
+    // if (resolvedAddresses.getAddresses().isEmpty()) {
```

Comment:
Make a decision on this?

Reply:
I was just being sloppy and forgot that in. I don't think we want to trigger a name resolution error as the current implementation does not do that either when receiving an empty address list.